libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/DrillIntersectionIgnoredAttributesSelect.tsx

```diff
@@ -0,0 +1,182 @@
+                    onApply(appliedSelection);
+                }
+            }}
+            renderBody={() => {
+                return (
+                    <InvertableSelect
+                        className="gd-drill-intersection-ignored-attributes-select"
+                        onSelect={(items, isInverted) => {
+                            setSelection(items);
+                            setIsInverted(isInverted);
+                        }}
+                        items={options}
+                        getItemTitle={(item) => item.title}
+                        getItemKey={(item) => item.id}
+                        isInverted={isInverted}
+                        selectedItems={selection}
+                        totalItemsCount={options.length}
+                        renderItem={({ title, item, onSelect, onDeselect, isSelected }) => {
+                            const isDisabled =
+                                item.type === "date" &&
+                                drillTargetType === DRILL_TARGET_TYPE.DRILL_TO_DASHBOARD;
+
+                            let IconComponent: React.ComponentType<IIconProps> = () => null;
+                            if (item.type === "date") {
+                                IconComponent = Icon.Date;
+                            } else if (item.type === "attribute") {
+                                IconComponent = Icon.Attribute;
+                            }
+                            return (
+                                <InvertableSelectItem
+                                    title={title}
+                                    renderOnly={() => <></>}
+                                    isDisabled={isDisabled}
+                                    isSelected={!isDisabled && isSelected}
+                                    icon={
+                                        <IconComponent
+                                            className={cx(
+                                                "gd-drill-intersection-ignored-attributes-select-item-icon",
+                                                `gd-drill-intersection-ignored-attributes-select-item-icon-${item.type}`,
+                                            )}
+                                        />
+                                    }
+                                    onClick={() => {
+                                        if (isDisabled) {
+                                            return;
+                                        }
+                                        if (isSelected) {
+                                            onDeselect();
+                                        } else {
+                                            onSelect();
+                                        }
+                                    }}
+                                />
+                            );
+                        }}
+                        renderStatusBar={() => <></>}
```

Comment:
is this necessary? did you need something like empty jsx?

Reply:
I didnt want to change typings and it's JSX.Element